drv_cfg.go

```diff
@@ -83,27 +97,33 @@ func DrvCfgQueryGUID() (string, error) {
 
 	opCode := _IO(_IOCTLBase, _IOCTLQueryGUID)
 
-	buf := [1]ioctlGUID{}
+	var buf ioctlGUID
 	// #nosec CWE-242, validated buffer is large enough to hold data
-	err = ioctl(f.Fd(), opCode, uintptr(unsafe.Pointer(&buf[0])))
+	err = ioctlWrapper(syscaller, f.Fd(), opCode, &buf)
 	if err != nil {
 		return "", fmt.Errorf("QueryGUID error: %v", err)
 	}
 
-	rc, err := strconv.ParseInt(hex.EncodeToString(buf[0].rc[0:1]), 16, 64)
+	rc, err := strconv.ParseInt(hex.EncodeToString(buf.rc[0:1]), 16, 64)
+	if err != nil {
+		return "", fmt.Errorf("Failed to parse return code: %v", err)
```

Comment:
Error messages should start with a small letter.

Reply:
Fixed